umbraco-cloud/release-notes/2022-01-06-releasenotes.md

```diff
@@ -1,30 +1,77 @@
+You are now able to access a simpler project overview page for each of your projects. Initially are able to see basic information such as name, alias, **plan and project status**. In the future, we are likely to add more project related information that currently isn’t presented in the portal.
 
 ![New Project Overview Page](https://user-images.githubusercontent.com/93588665/149168523-088b58f1-5a04-43ff-9ac5-f30b62c74e4e.gif)
 
+### Specify portal project roles on project invites
+
+When inviting a new team member to a project it is now possible **to specify the project roles directly in the project invite**. Often you want the team member to be assigned a different role for each environment. This is now supported from the start in the project invite. A user can now, for example, be assigned with the Admin role on the development environment while being limited to the Write role on the Live environment. All helping to make it faster and easier to set up the correct permissions for team members.
+
+![ProjectInvite\_v1](https://user-images.githubusercontent.com/93588665/150125691-cb846cbc-ad7f-4135-9a48-5de640776e62.gif)
+
+### Improved available upgrade notifications
+
+We have reworked a couple of the criteria that triggers **notifications** for an **available upgrade**. Now you will get a notification on Starter projects without a development environment as seen below.
+
+<figure><img src="https://user-images.githubusercontent.com/93588665/150126101-2a5b9de9-b5b8-4091-9e13-801eff1f8f6a.png" alt=""><figcaption></figcaption></figure>
+
+Another change is that the upgrade banner will no longer be displayed on projects with one or more environments already upgraded.
+
+### Baseline loading efficiency in create wizard
+
+The _Create new Cloud project wizard_ has in the last weeks undergone some **performance improvements**. The most significant of these is the loading of potential baselines for users with more than 50 cloud projects. Previously this could be time-consuming, this has now been updated to loading asynchronously.
+
+We have ensured that the potential baselines are loaded much faster to ensure a **better user experience** and less wasted time before you can start working on a new cloud project. ![BaselineAsyncLoad\_v1](https://user-images.githubusercontent.com/93588665/150125758-3fcb5664-f0b4-4bee-926e-ecbbfb113a09.gif)
+
+### [Blob storage connection info](../set-up/media/connect-to-azure-storage-explorer.md)
+
+The blob storage connection info for a project was previously only displayed in Kudu which was not convenient or easy to find. A developer should spend as much time as possible developing fantastic solutions and less time in Kudu. Therefore this connection info is now easy to _**copy directly from the portal**_ and ready to post into _Azure Blob Explorer_ whenever needed.
```

Comment:
It complains about the use if easy, however in this case it is used to convey the meaning so I think it should stay.